wordpress-mobile/gutenberg-mobile
# Update tests for Xcode 11/iOS 13

Fix CI tests for Xcode 11 on the server and locally.

To test:
 - Run tests locally
 - For iOS tests to run locally make sure you have Xcode 11.2 installed locally, and the iPhone 11 simulator installed to.
 - Check the tests on the server.

Update release notes:
